src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.ios.js

```diff
@@ -76,7 +79,9 @@ function CreatePollContainer({
       <TouchableOpacity style={S.polldurationbutton} onPress={() => setIsDurationModalShown(true)}>
         <View style={S.row}>
           <Text style={S.fillparenttext}>Duration</Text>
-          <Text style={S.polldurationbuttontext}>{getDurationTimeText(selectedtime)}</Text>
+          <Text style={[S.polldurationbuttontext, {backgroundColor: dynamicColor.primary}]}>
+            {getDurationTimeText(selectedtime)}
+          </Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dynamic color is now being applied to the background of the duration text based on the `isAnonimity` prop. This looks good as long as the dynamic colors have been thoroughly tested to ensure they provide sufficient contrast against the text color for readability.


<!-- This is an auto-generated comment by OSS CodeRabbit -->